### PR TITLE
Feature/doc-691: Document new Comments success and error callbacks

### PR DESCRIPTION
**Pull request by SimonFc**
Related Ticket: TINY-6345

Description of Changes: Documented new onError and onSuccess callbacks added to Comments.

Pre-checks:
- [x] Branch prefixed with `feature/` or `hotfix/`
- [x] <s>`_data/nav.yml` has been updated (if applicable)</s>
- [x]  <s>Files has been included where required (if applicable)</s>
- [x]  <s>Files removed have been deleted, not just excluded from the build (if applicable)</s>
- [x] (New product features only) Release Note added

Review:
- [x] Documentation Team Lead has reviewed
- [x] Product Manager has reviewed

### CODE REVIEW

File: release-notes/release-notes56.md [hunk 1]
```diff
@@ -40,6 +40,20 @@ The {{site.productname}} 5.6 release includes an accompanying release of the **<
 
 For information on the <<Premium Plugin Name>> plugin, see: [<<Premium Plugin Name>> plugin]({{site.baseurl}}/plugins/<<Premium Plugin Name>>/).
 
+### Comments 2.3.0
+
+The {{site.productname}} 5.5 release includes an accompanying release of the **Comments** premium plugin.
```
Comment (lnewson) [inline release-notes/release-notes56.md:45]: `5.6`?

**Review by lnewson — APPROVED** [review]

File: advanced/configuring-comments-callbacks.md [hunk 2]
```diff
@@ -52,7 +52,11 @@ The done callback needs to take an object of the form:
 
 ```js
 {
-  conversationUid: string // the new conversation uid
+  conversationUid: string, // the new conversation uid
+  // optional error callback which will be executed in case the new conversation could not be created
```
Comment (tylerkelly13) [inline advanced/configuring-comments-callbacks.md:56]:
```suggestion
  // Optional error callback which will be run if the new conversation could not be created
```

File: advanced/configuring-comments-callbacks.md [hunk 3]
```diff
@@ -52,7 +52,11 @@ The done callback needs to take an object of the form:
...
+  onError: function (err) { ... },
+  // optional success callback which will be executed when the new conversation is successfully created
```
Comment (tylerkelly13) [inline advanced/configuring-comments-callbacks.md:58]:
```suggestion
  // Optional success callback which will be run when the new conversation is successfully created
```

File: release-notes/release-notes56.md [hunk 4]
```diff
@@ -40,6 +40,20 @@ The {{site.productname}} 5.6 release includes an accompanying release of the **<
 
 For information on the <<Premium Plugin Name>> plugin, see: [<<Premium Plugin Name>> plugin]({{site.baseurl}}/plugins/<<Premium Plugin Name>>/).
 
+### Comments 2.3.0
+
+The {{site.productname}} 5.6 release includes an accompanying release of the **Comments** premium plugin.
+
+**Comments** 2.3.0 introduces `onError` and `onSuccess` callbacks for the `tinycomments_create` "done" function
```
Comment (tylerkelly13) [inline release-notes/release-notes56.md:47]:
```suggestion
**Comments** 2.3.0 introduces `onError` and `onSuccess` callbacks for the `tinycomments_create` "done" function.
```

File: release-notes/release-notes56.md [hunk 5]
```diff
@@ -40,6 +40,20 @@ The {{site.productname}} 5.6 release includes an accompanying release of the **<
...
+
+**Comments** 2.3.0 also provides the following bug fixes:
+
+- Fixed nested and overlapping comments breaking the content.
+- Fixed long comments getting cut off.
+- Fixed scrollbar appearing in textarea when writing a comment.
```
Comment (tylerkelly13) [inline release-notes/release-notes56.md:53]:
```suggestion
- Fixed a scrollbar appearing in the textarea when writing a comment.
```

**Review by tylerkelly13 — APPROVED** [review]
LGTM. Just a few minor suggestions I'd like addressed.

File: advanced/configuring-comments-callbacks.md (hunk 3 above)
Comment (ashfordneil) [reply advanced/configuring-comments-callbacks.md:58]: How come we say "if" for failures and "when" for successes, is that deliberate?

**Review by ashfordneil — APPROVED** [review]

**Comment by tylerkelly13** [comment]
@ashfordneil not intentional. Now I'm overthinking this issue.

File: advanced/configuring-comments-callbacks.md (hunk 3 above)
Comment (tylerkelly13) [reply advanced/configuring-comments-callbacks.md:58]: @ashfordneil not intentional. Now I'm overthinking this issue.

**Review by EkimChau — APPROVED** [review]

**Pull request merged**